soh/soh/OTRGlobals.cpp

```diff
@@ -1637,6 +1637,10 @@ extern "C" GetItemEntry ItemTable_RetrieveEntry(s16 tableID, s16 getItemID) {
 extern "C" GetItemEntry Randomizer_GetItemFromActor(s16 actorId, s16 sceneNum, s16 actorParams, GetItemID ogId) {
     s16 getItemModIndex;
     RandomizerCheck randomizerCheck = OTRGlobals::Instance->gRandomizer->GetCheckFromActor(actorId, sceneNum, actorParams);
+    // if we got unknown check here, we don't need to do anything else, just return the ogId.
+    if (randomizerCheck == RC_UNKNOWN_CHECK) {
+        return ItemTable_RetrieveEntry(MOD_NONE, ogId);
```

Comment:
Still isn’t super clear to me how a third MOD_ table is going to interact with this (and the obtainability checks) but we can cross that bridge later

Reply:
Each mod will likely have their own obtainability checks, randomizer should be the only thing calling this function, and it needs to make a distinction between randomizer items and vanilla items, and all the `ogIds` should be vanilla items.